src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/config/DisableHystrixFeignConfiguration.java

```diff
@@ -2,10 +2,8 @@
 
 import feign.Feign;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Scope;
 
-@Configuration
 public class DisableHystrixFeignConfiguration {
```

Comment:
if I understand correctly:
removing annotation gives you flexibility to inject configuration to particular feign client, I guess Configuration files are scanned at the startup and that could be injected to case worker feign client